var/spack/repos/builtin/packages/py-mappy/package.py

```diff
@@ -14,4 +14,5 @@ class PyMappy(PythonPackage):
 
     version('2.2', 'dfc2aefe98376124beb81ce7dcefeccb')
 
+    depends_on('python@:3.6.8')
```

Comment:
I'm assuming this is meant to convey `< 3.7` since there were some cython api changes then.  If that's the case, this would be better expressed as `python@:3.6` which means `3.6.x`.  The effect is the same but on the off chance there's another patch release as `3.6.9` then `@:3.6` covers that while `@:3.6.8` does not.